dts/bindings/usb/nxp,usbphy.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright  2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP USB High Speed Phy
+
+compatible: "nxp,usbphy"
+
+include: base.yaml
+
+properties:
+  D_CAL:
```

Comment:
Same comment I made on the devicetree source itself- can we use lowercase property names here, with dashes?